docs/guide/site_factory.md

```diff
@@ -0,0 +1,160 @@
+# Site Factory
+
+Site Factory is site management User Interface, integrated with Admin UI.
+ It allows you to configure sites without editing:
+
+- SiteAccess configuration
+- multisite configuration
+- multisite management
+
+After [clean installation](../getting_started/install_ez_enterprise.md) the Site Factory will be disabled.
+This results in the following message on the **Site** tab:
+"There is a design configuration error, and you will not be able to create a new site. Please update the configuration."
+If you plan to use Site Factory you need to enable and configure it.
+
+To enable or disable Site factory follow respectably:
+
+- [Enable Site Factory section](#enable-site-factory)
+- [Disable Site Factory section](#disable-site-factory)
+
+## Enable Site Factory
+
+To enable Site Factory you need to change enabled to `true` in `vendor/ezsystems/ezplatform-site-factory/src/bundle/Resources/config/site_factory.yaml`.
+
+### Configure designs
+
+Next you can configure it, add empty SiteAccess groups in `config/packages/ezplatform.yaml`:
+
+```yaml
+ezplatform:
+    siteaccess:
+        list: [site, pl, de]
+        groups:
+            site_group: [site, pl, de]
+            sf_group_1: []
+            sf_group_2: []
+            sf_group_3: []
+            
+    system:
+        sf_group_1:
+    
+        sf_group_2:
+    
+        sf_group_3:
+```
+
+Next, uncomment `'@EzSystems\EzPlatformSiteFactory\SiteAccessMatcher': ~` SiteAccess matcher  in `ezplatform.siteaccess.siteaccess`.
+ 
+Add `ezdesign` configuration and configure designs for empty SiteAccess groups:
+
+```yaml
+ezplatform:
+
+...
+
+ezdesign:
+    design_list:
+        example_1: [example_1_template]
+        example_2: [example_2_template]
+        
+ezpublish:
+    system:
+        sf_group_1:
+            design: example_1
+        sf_group_2:
+            design: example_2
+        sf_group_3:
+            design: example_3
+```
+
+The `ezdesign` defines templates for your sites, so remember to add them before continuing.
+
+### Add templates configuration
+
+Create `config/packages/ez_platform_site_factory.yaml`.
+It will connect SiteAccesses with your templates.
+Inside add thumbnails and names for your templates:
+
+```yaml
+ez_platform_site_factory:
+    templates:
+        ez_site1:
+            siteaccess_group: sf_group_1
+            name: example_site_1
+            thumbnail: /assets/ezplatform/build/images/example-1-icon.png
+        ez_site2:
+            siteaccess_group: sf_group_2
+            name: example_site_2
+            thumbnail: /assets/ezplatform/build/images/example-2-icon.png
+```
+
+### Define domains 
```

Comment:
Maybe worth adding that at this point (before adding domains) you can already use/try out site factory in the Back Office?

Reply:
added in section above